assets/jsx/bagger.jsx

```diff
@@ -17,8 +17,19 @@ class Bagger extends React.Component {
     }
 
     componentDidMount() {
-        const { dispatch, uploader: {accessKeyId, secretAccessKey, bucket, region}} = this.props
-        dispatch(BagActions.testConfiguration(accessKeyId, secretAccessKey, bucket, region))
+        const { dispatch } = this.props
+        const b = document.getElementById('bagger');
+        if (b.dataset.accessKeyId !== undefined) { // TODO: any? all? add fine grain actions?
```

Comment:
I feel like we should definitely require the access and secret keys, bucket, and region to have a value defined before doing anything. We should be careful not to require those to be changed since it's easy to imagine that we'd want to share links which would e.g. prefill the region / bucket / prefix.